generators/bdk/python/templates/ext-app/config.yaml.ejs

```diff
@@ -0,0 +1,11 @@
+host: <%= host %>
```

Comment:
Could we reuse the same `config.taml.ejs` template across different generators?

Reply:
Legacy projects use json config files and the java BDK2.0 ext app has a config file with a slightly different format (same config as the python config but inside a `bdk` field). There is an include option in ejs: https://ejs.co/#features but it means we will have the wrong indentation.